### PR TITLE
feat: Add support for rhel80-s390x

### PR DESCRIPTION
**Pull request by kryptonite303**
### Description

Adds support for rhel80-s390x boxes

### CODE REVIEW

File: tasks/setup-rhel80-s390x.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+- name: Copy over systemd template for node_exporter
```
Comment (jasonwalsh) [inline tasks/setup-rhel80-s390x.yml:2]: These tasks seem to be duplicated in `setup-rhel81-power8.yml`. Can we have a generic `setup-RedHat.yml` task list?

Comment (kryptonite303) [reply tasks/setup-rhel80-s390x.yml]: I combined the tasks into 1 file

File: tasks/main.yml [hunk 2]
```diff
@@ -38,9 +38,16 @@
   when:
     - ansible_distribution == "MacOSX"
 
-- name: Setup systemd for node_exporter on RHEL
-  include_tasks: setup-rhel81-power8.yml
+- name: Setup systemd for node_exporter on rhel81-power8
+  include_tasks: setup-rhel.yml
   when:
     - ansible_distribution == "RedHat"
     - ansible_architecture == "ppc64le"
```
Comment (jasonwalsh) [inline tasks/main.yml:45]: one last nit: is it possible to do something like `ansible_architecture in ["ppc64le", "s390x"]`?

Comment (jasonwalsh) [reply tasks/main.yml:45]: If so, you can probably rename the `include_tasks` directive to `"Setup systemd for node_exporter ({{ansible_architecture}})"`

Comment (kryptonite303) [reply tasks/main.yml]: Putting the architectures into an array worked

**Pull request merged**